src/DynamoRevitIcons/RevitNodesImages.resx

```diff
@@ -9702,6 +9702,464 @@
+        mbH80cdMVlrL+SjF0/qPd/wHexftIqV0d6QYjRSjkWI0UoxGitFIkYzjeASDo67nclB9iBQJTPuqHnw9
+        l3o9jxTJlKmFfx3iA1yOAONkNGg7cLK6RooEppvk0JqfC7WeIUUyZWqhe4jPp8cN2IIVeAGvYO3mqGXM
+        0zUEFn4DCXyD9zLOtZvnHLWz+bqFgDa4JoS/+qFobLrifbhauI3z9gvBKxbPQcqY286GfB3yfRgzxLmQ
+        g/1ZCGABON6WueeC7U5+VrqFMEqDrwIbGf5oyGV3uocgaMA3g6wNN3dVk24hoN3/zyo0BAxLwO+HZaU3
+        haAPzPueYHNwE6KVphAEpl+3r4Wp9aRIahPqna/n0hqCx7F3nKp6NqoPkWI0UoxGitFIMRopRiPFWNLi
+        B8aqNPyR+v/1AAAAAElFTkSuQmCC
+</value>
+  </data>
+  <data name="Revit.Elements.Space.Volume.Large" type="System.Drawing.Bitmap, System.Drawing" mimetype="application/x-microsoft.net.object.bytearray.base64">
+    <value>
+        iVBORw0KGgoAAAANSUhEUgAAAGEAAABhCAYAAADGBs+jAAAABGdBTUEAALGPC/xhBQAAAAlwSFlzAAAL
+        EgAACxIB0t1+/AAAA2tJREFUeF7t3DFuE0EYBeBcITdAOQESNRJFrkCfipouEhWlr5AutYXEAdLkABRc
+        IBInCEdY3m9mrMnwNsy/u/o9633Fp4Q3s2Z2nmOvLdkXwzDIidFQYtFQYtFQYtFQYtFQYtFQYtFQYtFQ
+        YtFQYtFQYtFQYtFQYtFQYtFQYtFQYtFQYtHQa7/ff4Bhq+r98KKhFxaSS7CfHvfws8rW5DNgC/i+tKKh
+        V1oQfuXjY3DMV3hkY2sw9bxrNPSauhgcoxKAhl5TF5OOu2FjazD1vGs09FpqMWujEjqgEjqgEjqgEjqg
+        EjqgEjpwNiWsubwuS4DHjM0rpfmj6vklNr9Uz6+xY0r1/BLGbyCfp73vhZjPbUVDLywkl2BvQxyweYwd
+        V2c9w3rtXPN52huQiPncVjT0SgvDr3z8NUucxKnMOe8SDb1UAh9vRUMvlcDHW9HQSyXw8VY09FIJfLwV
+        Db2WWszaqIQOqIQOqIQOqIQZ3n/5fg0D8cTmj1EJM2CzP1Wbnz2z+WNUwkzY8I/Jcy4A3rG5Y1TCArDp
+        P1IBZsfmvOZsSjhVeXavLwrIHtjcMSphAdh0e4K+hfIv4orNZTZZAjbI7r3XVWYb+SJL+VUa++dxHtkl
+        7JO6hObnhc2VgM2xJ9G8UbZpVshTkdkTq82xDX5IWWbzjpuL38eujrZ5ieoood7YfFVTZ+W9uh67TLfF
+        irJ/b/PqaEYJU9yS27WHrObngZJK+Pt4bvdodpVzBza2KzJj/xm9/SlUAh7/i7H6Iehwz8bP+u0J1yXo
+        /5xNCa1sA6sNPV4R1WNFrhKWZBtYbahKKKkEPt6Khl4qgY+3oqFXUAl2NVRuqEooBZVQvu1sm3544ZXG
+        7K2HvNF31XG5oMMr6nJsrs2V0KOzKWHN5amEDqiEDqiEDqiEDqiEDqiEDqiEDvRWwjdbDJz7JzffwvET
+        qrADDPH5rWjolRcDrk9urg3O7U0+x2T9X7+2dkudNw29VAIfb0VDL5XAx1vR0Esl8PFWNPRSCXy8FQ29
+        VAIfb0VDr6mLwTF2mafvRWWh19TF4BiVADT0mroYHKMSgIZeeTG2oU6/4HeVrUlXXzpVv5zfFLYnHjSU
+        WDSUWDSUWDSUWDSUWDSUWDSUWDSUWDSUWDSUWDSUWDSUWDSUWDSUWDSUWDSUSMPFH7gQV/wFQICRAAAA
+        AElFTkSuQmCC
+</value>
+  </data>
+  <data name="Revit.Elements.Space.Volume.Small" type="System.Drawing.Bitmap, System.Drawing" mimetype="application/x-microsoft.net.object.bytearray.base64">
+    <value>
+        iVBORw0KGgoAAAANSUhEUgAAACEAAAAhCAYAAABX5MJvAAAABGdBTUEAALGPC/xhBQAAAAlwSFlzAAAL
+        EgAACxIB0t1+/AAAAR1JREFUWEfdlr0NwjAQRrNCNkBMwAJIFAxAkz4VKyBRUWYE6KjTQE/DAKzACqxg
+        vpPO0sk5fhzHh0LxdPgJ7j47RkrhnPs5qrRGldao0hpVWqNKa1RJtG17BDvBnpEuBuq30GZ1hId+GKxn
+        hHQxUIDkEKmMOsQZ1IFTm3yCA6TfCXa9QhAc5E9CfMN8eyrBNPTWIW7AgUr6wUK8aiLB8DWHWEqfJQSG
+        bMAF0M4bri3XB8h/Ehhy4GEUhHbu6wrY3AkMol3fgT9+9TF4coago6/8cF+17w8WQjbBMPorlv4z185j
+        8GQJEcuoQ1xThkrQpwZNnxBvL2YsFCA5BNYTQroY+oYI3zFpHboY4t8nLFGlNaq0RpXWqNIaVdriiidp
+        S0RiDjabCgAAAABJRU5ErkJggg==
+=======
```

Comment:
I think here missing end tag </value> </data>, I can't build this solution successfully.